pkg/storage/versioner/versioner.go

```diff
@@ -43,8 +40,10 @@ func (a APIObjectVersioner) UpdateObject(obj runtime.Object, resourceVersion uin
 	return nil
 }
 
-// UpdateList implements Versioner
-func (a APIObjectVersioner) UpdateList(obj runtime.Object, resourceVersion uint64) error {
+// UpdateList sets the resource version into an API list object. Returns an error if the object
+// cannot be updated correctly. May return nil if the requested object does not need metadata
+// from database.
+func UpdateList(obj runtime.Object, resourceVersion uint64) error {
```

Comment:
SetStorageMetadata, if we're changing names, but that would make this into a much more invasive change, I guess.